src/renderer.cpp

```diff
@@ -233,6 +233,8 @@ void Renderer::updateRendererShaderPaths()
     fragmentShadowMapPath += ".smooth" + shaderSuffix + ".frag";
     vertexFieldShadowPath += ".smooth" + shaderSuffix + ".vert";
     fragmentFieldShadowPath += ".smooth" + shaderSuffix + ".frag";
+    vertexBlitPath += ".smooth" + shaderSuffix + ".vert";
+    fragmentBlitPath += ".smooth" + shaderSuffix + ".frag";

```

Comment:
Technically it should not matter because when we blit we don't use the vertex colors at all, but shouldn't this technically be `.flat` instead?

Reply:
Changed it to `.flat`